documentation/CLI-for-JFrog-Artifactory.md

```diff
@@ -893,6 +893,7 @@ By default, the command collects the files from the local file system. If you'd
 | --recursive       | \[Default: true\]<br><br>When false, artifacts inside sub-folders in Artifactory will not be affected.                                                                                                                                                                                                                                                              |
 | --regexp          | \[Optional: false\]<br><br>\[Default: false\] Set to true to use a regular expression instead of wildcards expression to collect files to be added to the build info.This option is not supported when --from-rt is set to true.                                                                                                                                    |
 | --dry-run         | \[Default: false\]<br><br>Set to true to only get a summery of the dependencies that will be added to the build info.                                                                                                                                                                                                                                               |
+| --module          | \[Optional\]<br><br>Optional module name for the build-info.                                                                                                                                                                                                                                                                                                        |
```

Comment:
Optional module name for the build-info.
-->
Optional module name in the build-info for adding the depedency

(let's ensure to use the same phrasing for the embedded CLI help as well).